jupyterlab_gitsync/src/components/file.tsx

```diff
@@ -39,4 +42,4 @@ export class File extends React.Component<Props> {
   private _onResolveClick() {
     console.log('file resolved');
```

Comment:
no console.log please. this is why reviewers are hesitate to give approval on this, because it's easy to forget.